konnectivity-client/go.mod

```diff
@@ -5,18 +5,30 @@ go 1.17
 // Prefer to keep requirements compatible with the oldest supported
 // k/k minor version, to prevent client backport issues.
 require (
```

Comment:
@tallclair PTAL, I'm not confident about this.

(Trying to stay as backward compat with k/k as possible, without the replace I end up getting protobuf 1.5.4)

Reply:
I fixed this in latest snapshot.